src/services/bundle-verifier.ts

```diff
@@ -0,0 +1,20 @@
+import { CLIError } from '@oclif/errors'
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+
+export default class BundleVerifier {
```

Comment:
1. don't use default export as @Dv92 suggested
2. I would call it BundleService, we may add other functions later